### ISSUE TITLE
initgroups unsupported

### ISSUE THREAD

**Issue by russellvt** [trlinkin/puppet-nsswitch#18]
Please add [initgroups](http://man7.org/linux/man-pages/man5/nsswitch.conf.5.html) to the supported parameters.

**Comment by trlinkin** [comment]
Yeah, ok! Next chance I get to do a little work. If you'd like to take a shot at a PR, that's also welcome :)

**Comment by bp85** [comment]
@trlinkin any updates on merging `initgroups` branch into main?
